README.md

```diff
@@ -20,8 +18,6 @@ To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-Basics of HTML, CSS, git,github, linter
-
 ### Setup
 
 ### Install
```

Comment:
- [ ] Kindly add the project setup for this project. Kindly understand that making your project easily to navigate is a good plus point for a professional project. You can use this sample as a guide and add other details 😄  
![image](https://user-images.githubusercontent.com/63473584/176757007-17b1a79b-55b3-4c38-90fc-a397fdfd29f1.png)